charts/delphai-keda/templates/ingress.yml

```diff
@@ -54,25 +49,32 @@ spec:
     credentials: true
     max_age: "86400"
   {{ end }}
-{{ end }} {{/* if and $.Values.http.enabled $.Values.http.public */}}
 
-{{ if and $.Values.grpc.enabled $.Values.grpc.public }}
-{{ if not (lookup "v1" "Service" "ambassador" "ambassador") }}
+{{ if eq $.Values.subdomain "api" }}
 ---
 apiVersion: getambassador.io/v2
-kind: Host
+kind: Mapping
 metadata:
-  name: {{ $.Release.Name }}-domain{{ $domain_index }}-grpc
+  name: {{ $.Release.Name }}-domain{{ $domain_index }}-http-prefix
   namespace: {{ $.Release.Namespace }}
 spec:
-  hostname: "{{ $.Release.Name }}.grpc.{{ $domain }}"
-  acmeProvider:
-    authority: none
-  requestPolicy:
-    insecure:
-      action: Route
-{{ end }} {{/* if not (lookup "v1" "Service" "ambassador" "ambassador") */}}
+  prefix: "/service/{{ $.Release.Name }}/"
+  service: "{{ $.Release.Name }}:{{ if $.Values.http.publicWithOAuth }}{{ $.Values.http.oAuthProxyPort }}{{ else }}{{ $.Values.http.port }}{{ end }}"
+{{ if eq $.Values.delphaiEnvironment "review" }}
+  host: "^app.*\\.{{ $domain }}$"
```

Comment:
Envoy implies `^` and `$`, but I added it here to be more verbose.
It works as before

Reply:
I mean .* -> .*?
but it's not that much important. just a good habbit

Reply:
I know what's greedy.
I can change if you prefer